core/src/main/java/org/infinispan/xsite/OfflineStatus.java

```diff
@@ -47,10 +47,16 @@ public OfflineStatus(TakeOfflineConfiguration takeOfflineConfiguration, TimeServ
 
    public synchronized void updateOnCommunicationFailure(long sendTimeMillis) {
       if (!recordingOfflineStatus) {
+         if (trace) {
+            log.trace("Start recording failures");
```

Comment:
Is this really needed?

Reply:
removed all traces. It was for debugging a test xD
:+1: